.github/workflows/builds.yml

```
@@ -14,6 +16,68 @@ jobs:
       - name: Run built Diffkemp
         run: result/bin/diffkemp --help
 
+  local-build:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - build-type: release
+            diffkemp-bin: diffkemp
+          - build-type: development
+            diffkemp-bin: bin/diffkemp
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Python
+        uses: actions/setup-python@v5
+      - name: Install Dependencies
+        run: |
+          sudo apt update
+          sudo apt install cmake ninja-build libgtest-dev
+          pip install -r requirements.txt
+      - name: Install LLVM
+        run: |
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          sudo add-apt-repository "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-${{ env.llvm }} main"
+          sudo apt update
+          sudo apt install llvm-${{ env.llvm }}-dev clang-${{ env.llvm }}
+          echo "/usr/lib/llvm-${{ env.llvm }}/bin" >> $GITHUB_PATH
+      - name: Release build and install of DiffKemp
+        if: ${{ matrix.build-type == 'release' }}
+        run: |
+          cmake -S . -B build -GNinja -DCMAKE_BUILD_TYPE=Release
+          sudo ninja -C build install
+          pip install .
+          sudo install -m 0755 bin/diffkemp /usr/bin/diffkemp
+      - name: Development build of DiffKemp
+        if: ${{ matrix.build-type == 'development' }}
+        run: |
+          cmake -S . -B build -GNinja
```

Maybe add explicit debug build type?
```suggestion
          cmake -S . -B build -GNinja -DCMAKE_BUILD_TYPE=Debug
```

---

Thanks, I added it, also I added it to the nix development build.